src/button.tsx

```diff
@@ -21,19 +21,22 @@ export const BUTTON_MIN_WIDTH = 136;
 const transitionTiming = '0.3s cubic-bezier(0.77, 0, 0.175, 1)';
 
 const BORDER_PX = 1.5;
-const SPINNER_MARGIN_PX = 8;
+const ICON_MARGIN_PX = 8;
 const X_PADDING_PX = 16 - BORDER_PX;
 const Y_PADDING_PX = 12 - BORDER_PX;
 const X_SMALL_PADDING_PX = 12 - BORDER_PX;
 const Y_SMALL_PADDING_PX = 6 - BORDER_PX;
+const ICON_SIZE = 24;
+const SMALL_ICON_SIZE = 20;
+const SPINNER_SIZE = 24;
```

Comment:
```suggestion
const SPINNER_SIZE = 20;
```
Change this one to make the loading similar to icon size (icons have 2px padding inside, the real vector size is 20px)

Reply:
hmmm... are you sure? it was 24 before my changes

Reply:
Yes, I know, we was talking @aweell and I to make this little change to keep a smooth consistency across icons and spinners